src/rdb.c

```diff
@@ -1556,6 +1557,21 @@ int rdbSave(int req, char *filename, rdbSaveInfo *rsi, int rdbflags) {
     return C_OK;
 }
 
+void sendReplicationOffsetToReplicas(int req) {
+    listNode *ln;
+    listIter li;
+    listRewind(server.slaves, &li);
+    while((ln = listNext(&li))) {
+        client *replica = ln->value;
+        if ((replica->replstate == SLAVE_STATE_WAIT_BGSAVE_START) &&
+            (replica->slave_req == req) &&
```

Comment:
On second thought, I don't think we should support rdb-channel sync when the master uses disk. I may be wrong, but disk-based sync at the master side main advantage is that the background process is not dependent on the replica's responsiveness. This is useful for cases where replicas are slow and we want to avoid bgsave process COW (in exchange to larger COB). When used in such cases, rdb-channel sync will:
1. Use child process to write snapshot to disk.
2. **Use child process again** to write snapshot from disk to replicas sockets. 

RDB-channel will therefore degrade the performance of master disk base syncs. It is only used for that sync type, so I would like to remove this method entirely.